client/app/bundles/App/GettingStarted/Theming.md

```diff
@@ -1,3 +1,103 @@
+Terra UI provides theming capabilities in various components via [CSS custom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/--*).
+
+### Terra default theme
+Terra UI provides a default theme for components via [CSS custom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/--*).
+For teams that do not have additional theming needs outside of the default terra theme, we recommend using the [PostCSS loader for webpack ](https://github.com/postcss/postcss-loader/) along with the [CustomProperties plugin](https://github.com/postcss/postcss-custom-properties) to compile CSS custom properties down to static values. This ensures the theme styles render in modern browsers as well as those which do not support CSS custom properties.
+
+
+__Recommend webpack setup for this use case:__
+```js
+// Require the PostCSS Custom Properties plugin
+const CustomProperties = require('postcss-custom-properties');
+
+...
+
+// In your webpack config
+{
+  loader: 'postcss-loader',
+  options: {
+    plugins() {
+      return [
+        CustomProperties(),
+      ];
+    },
+  },
+},
+```
+
+### First class Cerner UX themes
+Cerner UX is in the process of designing first-class themes for Terra UI. Terra UI currently supports the following first-class custom themes:
+* `consumer`
+
+### Changing the default theme
+For teams that want to use one of the first class Cerner UX themes as their default theme, we recommend using the [PostCSS loader for webpack ](https://github.com/postcss/postcss-loader/) along with the [CustomProperties plugin](https://github.com/postcss/postcss-custom-properties) in addition to the [webpack sass-loader](https://github.com/webpack-contrib/sass-loader) to specify a default theme.
+
+The `$default-theme` variable should be a single string value of the name of a first class Cerner UX theme. e.g. `$default-theme: ux-specified-theme-1;`
+
+This will replace the default terra CSS custom properties values with those specified in the default theme and compile them down to static values.
+
+__Recommend webpack setup for this use case:__
+```js
+// Require the PostCSS Custom Properties plugin
+const CustomProperties = require('postcss-custom-properties');
+
+...
+
+// In your webpack config
+{
+  loader: 'postcss-loader',
+  options: {
+    plugins() {
+      return [
+        CustomProperties(),
+      ];
+    },
+  },
+  {
+    loader: 'sass-loader',
+    options: {
+      data: '$default-theme: ux-specified-theme-1;',
+    },
+},
+```
+
+### Using the theme provider to dynamically switch between themes
+Terra UI offers a theme provider component to theme various Terra UI components via CSS custom properties a.k.a CSS variables. This is accomplished by setting a CSS class which contains defined CSS custom properties for the specified theme on the DOM element that wraps the children rendered by the theme provider.
```

Comment:
should we add  

> a.k.a CSS Variables 

at the top when we first introduce it?

Reply:
Updated: https://github.com/cerner/terra-ui.com/pull/53/commits/db2238d6f8baef77144add2e1df6248b460be568